pycm/pycm_curve.py

```diff
@@ -63,12 +67,27 @@ def lambda_fun(x): return threshold_func(
                 for item in CURVE_PARAMS:
                     data_temp[item].append(getattr(cm, item)[c])
             self.data[c] = data_temp
-        self.plot_x_axis = None
-        self.plot_y_axis = None
+        self.auc = None
+        self.plot_x_axis = self.data[self.classes[0]]["FPR"]
```

Comment:
Here, I suggest saving the parameter names in `self.plot_x_axis` and `self.plot_y_axis` instead of vectors.
To support the multi-class mode that I mentioned in another comment.